LEAF_Request_Portal/api/controllers/NotesController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/*
+ * As a work of the United States government, this project is in the public domain within the United States.
+ */
+
+require '../sources/Note.php';
+
+if (!class_exists('XSSHelpers'))
+{
+    include_once dirname(__FILE__) . '/../../../libs/php-commons/XSSHelpers.php';
+}
+
+class NotesController extends RESTfulResponse
+{
+    /**
+     *
+     * @var array
+     */
+    public $index = array();
+
+    /**
+     *
+     * @var int
+     */
+    private $API_VERSION = 1;
+
+    /**
+     *
+     * @var \Note
+     */
+    private $note;
+
+    /**
+     *
+     * @param \Db $db
+     * @param \Login $login
+     * @param \DataActionLogger $dataActionLogger
+     *
+     * Created at: 10/7/2022, 9:45:22 AM (America/New_York)
+     */
+    public function __construct(\Db $db, \Login $login, \DataActionLogger $dataActionLogger)
+    {
+        $this->note = new Note($db, $login, $dataActionLogger);
+    }
+
+    public function get($act)
```

Comment:
Need to clean up these functions.  Can you give them the full tune-up:  type-hinting, defaults, return types, single returns?

Reply:
I was going to do that but because they are inherited from RestfulResponse there is a conflict because they don't match. Would you like me to do the complete overhaul including updating the RestfulResponse?